packages/manager/apps/key-management-service/src/pages/dashboard/generalInformations/updateName/OkmsNameUpdateModal.tsx

```diff
@@ -0,0 +1,39 @@
+import { UpdateNameModal } from '@ovh-ux/manager-react-components';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useUpdateOkmsName } from '@/data/hooks/useUpdateOkmsName';
+import { useKMSServiceInfos } from '@/data/hooks/useKMSServiceInfos';
+
+const OkmsNameUpdateModal = () => {
+  const { t } = useTranslation('key-management-service/dashboard');
+  const { okmsId } = useParams();
+  const { data: okmsServiceInfos, isLoading } = useKMSServiceInfos(okmsId);
+  const navigate = useNavigate();
+  const { updateKmsName, isPending } = useUpdateOkmsName({
+    okmsId,
+    onSuccess: () => {
+      navigate('..');
+    },
+    onError: () => {
+      navigate('..');
+    },
```

Comment:
I don't know the requirements for this ticket, but shouldn't we display the error on the modal to let the user retry ?

Reply:
And you can provide the error prop to UpdateNameModal that will display it automatically